xen/arch/arm/domain.c

```diff
@@ -698,6 +702,27 @@ int arch_sanitise_domain_config(struct xen_domctl_createdomain *config)
     return 0;
 }
 
+#ifdef CONFIG_HOST_DTB_EXPORT
```

Comment:
Why it is living in the middle of `domain.c`?  This file is for low-level domain state manipulation

Reply:
`bootfdt.c` maybe?